Assignments/stu1/api.js

```diff
@@ -25,7 +25,12 @@ app.use(cors());
 // GET /ex1/persons/
 //
 
-// Ex. 1: TODO ...
+app.get('/ex1/persons/', cors(corsOptions), async (req, res) => { 
+    let personId = req.params['persons'];
```

Comment:
The URL for this endpoint is: `/ex1/persons/`  Therefore we do not need ` let personId = req.params['persons'];` and we do not need to pass `personId` into `dataAccess.getPerson(personId)`